db/migrations/public/20231127230416_create-workflows-view.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const viewName = 'workflows'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      view.as(knex('charge_version_workflows').withSchema('water').select([
+        'charge_version_workflow_id AS id',
+        'licence_id',
+        'created_by',
+        'approver_comments',
+        'status',
+        'data',
+        'licence_version_id',
+        'date_deleted',

```

Comment:
What do you think about making this consistent with the other fields?

```suggestion
        'date_deleted AS deleted_at',

```